### PR TITLE
APP-958 - Updated README to add an example for MessageML v2

### PR DESCRIPTION
**Pull request by luanapp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,225 +11,165 @@ If you are a JIRA admin user, you can configure a JIRA WebHook (as shown in the
...
-```
-* Message rendered on Symphony (since this sample does not have a "description" tag, it will show as "n/a")
-
-![Issue Updated rendered](src/docs/sample/sample_issue_updated_rendered.png)
-
-### Comment Created
 
-* [Comment Created JSON file](src/docs/sample/jiraCallbackSampleCommentAdded.json)
-* Message ML generated by the parsers
+##### MessageML and JSON entity supported by the Universal Webhook Integration (MessageML v2)
```
Comment (robson-sanchez) [inline README.md]: Universal Webhook ?

File: README.md [hunk 2]
```diff
@@ -11,225 +11,165 @@ If you are a JIRA admin user, you can configure a JIRA WebHook (as shown in the
...
 
 ## What formats and events it supports and what it produces
-Every integration will receive a message sent in a specific format (depending on the system it ingests) and will usually convert it into an "entity" before it reaches the Symphony platform. It will also, usually, identify the kind of message based on an "event" identifier, which varies based on the third-party system.
+Every integration will receive a message sent in a specific format (depending on the system it ingests) and will usually convert it into an internal format before it reaches the Symphony platform. It will also, usually, identify the kind of message based on an "event" identifier, which varies based on the third-party system.
 
-You can find more details about entities and the Symphony Message ML format [here](https://github.com/symphonyoss/App-Integrations-Core#the-message-ml-format).
+There are currently two implementation versions for the MessageML format, [V1](https://github.com/symphonyoss/App-Integrations-Core#the-message-ml-format) and [V2](https://symphonyoss.atlassian.net/wiki/display/WGFOS/MessageML+V2+Draft+Proposal+-+For+Discussion).
```
Comment (robson-sanchez) [inline README.md]: I believe this paragraph is not necessary once we're removing references from MessageML v1.

File: README.md [hunk 3]
```diff
@@ -11,225 +11,165 @@ If you are a JIRA admin user, you can configure a JIRA WebHook (as shown in the
...
-* Message rendered on Symphony (since this sample does not have a "description" tag, it will show as "n/a")
-
-![Issue Updated rendered](src/docs/sample/sample_issue_updated_rendered.png)
-
-### Comment Created
 
-* [Comment Created JSON file](src/docs/sample/jiraCallbackSampleCommentAdded.json)
-* Message ML generated by the parsers
+##### MessageML and JSON entity supported by the Universal Webhook Integration (MessageML v2)
+###### MessageML
```
Comment (robson-sanchez) [inline README.md]: You should add further information about MessageML v2 like Agent version that supports MessageML v2.

Example: https://github.com/symphonyoss/App-Integrations-Zapier#generated-symphony-message-and-json-entity-messageml-v2

**Review by robson-sanchez — CHANGES_REQUESTED** [review]
Small changes

File: README.md (hunk 2 above)
Comment (luanapp) [reply README.md]: Indeed!
Removed!

File: README.md (hunk 1 above)
Comment (luanapp) [reply README.md]: Oops
Copy error!
Fix'd

File: README.md (hunk 3 above)
Comment (luanapp) [reply README.md]: Since the Zapier text is well written, it was used in this section, with the name substitution this time ;-)

**Review by robson-sanchez — APPROVED** [review]

**Pull request merged**